### DISCUSSION TITLE
mihomo内核各种奇怪的问题

### DISCUSSION THREAD

**Discussion by ValeriaXYN** [MetaCubeX/mihomo#1115] [General]
客户端为Clash Nyanpasu，使用mihomo内核，开启系统代理时Steam无法下载，会显示无互联网连接，有时无法连接Github和知乎（这是最奇怪的因为知乎在规则里是直连）。切换Clash内核后问题马上就消失了。

**Comment by Victor2333** [comment]
提issue上配置和log再看看吧，听起来就是没配置好，和原生的clash有些地方还是有区别的
